web/packages/teleport/src/services/notifications/types.ts

```diff
@@ -24,19 +24,43 @@ export type FetchNotificationsResponse = {
    */
   notifications: Notification[];
   /**
-   * userNotificationsNextKey is the nextKey for the user-specific notifications list.
+   * nextKey is the next page keys for both lists (user-specific notifications & global notifications)
+   * separated by a comma, ie. "<user-specific notifications nextKey>,<global notifications nextKey>".
+   * If either one of these nextKeys is "end", it means we have reached the end of that list.
    */
-  userNotificationsNextKey: string;
-  /**
-   * globalNotificationsNextKey is the nextKey for the global notifications list.
-   */
-  globalNotificationsNextKey: string;
+  nextKey: string;
   /**
    * userLastSeenNotification is  the timestamp of the last notification the  user has seen.
    */
   userLastSeenNotification: Date;
 };
 
+/**
+ * UpsertLastSeenNotificationRequest is the request to upsert the timestamp of the latest notification that
+ * the user has seen.
+ */
+export type UpsertLastSeenNotificationRequest = {
+  /**
+   * time is the timestamp of the last seen notification.
+   */
+  time: Date;
+};
+
+/**
+ * UpsertNotificationStateRequest is the request made when a user updates a notification's state by marking it
+ * as "clicked" or "dismissed".
+ */
```

Comment:
whats the difference?

Reply:
"Clicked" just means that the user has already clicked on it and the notification will have a slightly dimmer colour in the list, "dismissed" hides the notification completely and the user will never see it again